tests/integration/cqlengine/query/test_named.py

```diff
@@ -280,6 +281,7 @@ def setUp(self):
 
     def tearDown(self):
         models.DEFAULT_KEYSPACE = self.default_keyspace
+        setup_connection(models.DEFAULT_KEYSPACE)
```

Comment:
I think calling this multiple times will leave dangling sessions (cqlengine.connection does not clean up by itself).